client/blocks/followers-count/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+import { getSelectedSite } from 'state/ui/selectors';
+
+class FollowersCount extends Component {
+	render() {
+		const { site, translate } = this.props;
+
+		if ( ! site || ! site.subscribers_count ) {
+			return null;
```

Comment:
I wonder if this component really needs to be a `connect()`ed component. Normally we would prefer to handle conditional rendering in the parent because a `null` render actually still keeps the component in memory even though it doesn't generate any HTML.

If we move control up to the parent…

```js
import { get } from 'lodash';

{ get( site, 'subscribers_count', 0 ) > 0 &&
	<FollowersCount siteSlug={ site.slug } count={ site.subscribers_count } />
}
```

…then we wouldn't have to return `null` and we wouldn't need the connect. It appears like that `site` object should already be available in the `stats-navigation` block.

Although I realize this is a block, I'm not entirely sure it needs to be. If we simply passed in the slug and count number then it would be much easier to write tests for. Granted, some of us do prefer to keep it as-is and handle the conditional render logic inside the component.